setup.py

```diff
@@ -13,7 +13,7 @@
 
 setup(
     name='sphinx-docfx-yaml',
-    version='1.2.73',
+    version='1.2.74',

```

Comment:
I consider this to be a bug that is being fixed, so I'm calling this a patch.

There are arguments to be made that this is enough of an enhancement to call this a minor version. 

Up to you @joelmartinez

Reply:
Yeah this is fine by me :)